week-1/1-exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,25 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+//Exercise 1//
+function percentProgram(firstNumber, secondNumber) {
+  let overallTotal = firstNumber + secondNumber;
+
+  let percentage = Math.floor((firstNumber / overallTotal) * 100);
```

Comment:
Running this exercise returns 65% for students and 34% for mentors.  The total is 99% not 100%, can you think why?